src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationNewWithTernaryOp.java

```diff
@@ -0,0 +1,43 @@
+/* Config:                                                                    //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration:   //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ * basicOffset = 4                                                            //indent:1 exp:1
+ * forceStrictCondition = true                                                //indent:1 exp:1
+ * lineWrappingIndentation = 8                                                //indent:1 exp:1
+ * tabWidth = 4                                                               //indent:1 exp:1
+ * throwsIndent = 8                                                           //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ */                                                                           //indent:1 exp:1
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+public class InputIndentationNewWithTernaryOp { //indent:0 exp:0
+    public Integer foo() { //indent:4 exp:4
+        boolean flag = true; //indent:8 exp:8
+        Integer result = flag ? //indent:8 exp:8
+                new Integer(1) : //indent:16 exp:16
```

Comment:
According to issue, they want 12 to be acceptable.
Please show an input matching user's request to produce no violations.

Then add or point to cases where formatting is not valid and how violation will appear.

Reply:
sorry i didn't get it.

Reply:
i have done something but something is wrong or maybe i don't exactly understand the regression report difference concept.

Reply:
i don't think i understand the concept really, i have added new test cases with violations after seeing it in generated report but it's still not working.

Reply:
> According to issue, they want 12 to be acceptable.

@rnveach could you please clarify this a little bit more

Reply:
please read https://github.com/checkstyle/checkstyle/pull/9766#discussion_r633143607